src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+    protected String from;
+    protected String to;
```

Comment:
Similarly, consider naming the variable 'from' and 'to' to something more descriptive, eg. startDate and endDate